source/core/bot.py

```diff
@@ -78,7 +81,7 @@ def init_pymongo_db():
     client = MongoClient(conf.properties["DB_PARAMS"]["ip"], conf.properties["DB_PARAMS"]["port"],
                          serverSelectionTimeoutMS=5000)
     try:
-        logger.debug(client.server_info())
+        logger.info(client.server_info())
```

Comment:
No. Debug. Because we don't need this information in logs.txt file if bot even not launch. In addition, server_info may contain confidential information (about machine)
```suggestion
        logger.debug(client.server_info())
```